workflow/controller/taskset.go

```diff
@@ -138,7 +138,7 @@ func (woc *wfOperationCtx) reconcileTaskSet(ctx context.Context) error {
 			if err != nil {
 				woc.log.Warnf("[SPECIAL][DEBUG] returning but assumed validity before")
```

Comment:
I think this logging is invalid if we continue.

Reply:
Removed this warning log.